recipe/build.sh

```diff
@@ -15,11 +20,34 @@ cd build
 cmake ${CMAKE_ARGS} \
   -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
   -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_LIBRARY_PATH="${PREFIX}" \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DLLVM_EXTERNAL_LIT="${BUILD_PREFIX}/bin/llvm-lit" \
+  -DMLIR_INCLUDE_DOCS=OFF \
+  -DMLIR_INCLUDE_TESTS=ON \
   -DLLVM_BUILD_LLVM_DYLIB=ON \
+  -DMLIR_INCLUDE_INTEGRATION_TESTS=ON \
+  -DLLVM_SHLIB_OUTPUT_INTDIR="${SRC_DIR}/build/lib" \
   -DLLVM_LINK_LLVM_DYLIB=ON \
   -DLLVM_BUILD_TOOLS=ON \
   -DLLVM_BUILD_UTILS=ON \
   -GNinja \
   ../mlir
 
-ninja
+cmake --build . -- -j${CPU_COUNT}
+#echo ${BUILD_PREFIX}
+#ls -la "${BUILD_PREFIX}/bin"
+# is PREFIX the env or host directory?
+#echo ${PREFIX}
+
+# this is where lit expects to find helper tools. Perhaps they should be put here while building llvm.
+cp ${PREFIX}/libexec/llvm/* ${PREFIX}/bin
```

Comment:
Same here, should we remove files copied in `${PREFIX}/bin` after the tests are run?

Reply:
done